### PR TITLE
Part4

### PR DESCRIPTION
**Pull request by daviddassau**
Firebase link: https://angular-contacts-db03c.firebaseapp.com

### CODE REVIEW

**Comment by daviddassau** [comment]
@zoeames @morecallan @rousell 🥇

File: javascripts/controllers/DetailCtrl.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+app.controller("DetailCtrl", function ($routeParams, $scope, ContactService){
+
+    $scope.contact = {};
+
+    console.log("ContactId", $routeParams.id);
+
+    const getContact = () => {
+        ContactService.getSingleContact($routeParams.id).then((results) => {
+            console.log("results.data", results.data);
```
Comment (morecallan) [inline javascripts/controllers/DetailCtrl.js]: Make sure to remove console logs from your PR.

**Review by morecallan — COMMENTED** [review]
@daviddassau - just need to clean up this code a bit, push back up, and we'll merge it!

File: javascripts/controllers/DetailCtrl.js (hunk 1 above)
Comment (daviddassau) [reply javascripts/controllers/DetailCtrl.js]: @morecallan @zoeames @rousell  GTG!!!

**Comment by daviddassau** [comment]
@morecallan All console.logs and commented out code have been removed!!!

**Pull request merged**